plugins/aws/s3/bucketEncryptionInTransit.js

```diff
@@ -86,5 +97,98 @@ module.exports = {
             }
         }
         callback(null, results, source);
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var putCall = this.actions.remediate;
+        var pluginName = 'bucketEncryptionInTransit';
+        var bucketNameArr = resource.split(':');
+        var bucketName = bucketNameArr[bucketNameArr.length - 1];
+
+        // find the location of the bucket needing to be remediated
+        var bucketLocations = cache['s3']['getBucketPolicy'];
+        var bucketLocation;
+
+        for (var key in bucketLocations) {
+            if (bucketLocations[key][bucketName]) {
+                bucketLocation = key;
+                var policy = bucketLocations[key][bucketName];
```

Comment:
declare policy outside of the for loop.

Reply:
how does it matter since we are breaking out of the loop immediately once we get the data needed.